daffodil-sapi/src/main/scala/org/apache/daffodil/sapi/Daffodil.scala

```diff
@@ -176,7 +179,7 @@ class Compiler private[sapi] () {
    *                  no namespace. Set to to NULL to figure out the namespace.
    */
   def setDistinguishedRootNode(name: String, namespace: String): Unit =
```

Comment:
Same questions as the Java API. Should we deprecate these, and can the var constructor paremters be made private.